docs/design/dockerfiles.md

```diff
@@ -146,6 +144,8 @@ as this is _not_ a volume.
 The user profile is updated to add the python binary directory to `PYTHONPATH` and
 the lib directory is appended to `LD_LIBRARY_PATH`.
 
+**REMOVE_OPTIX** _(Default: `false`)_: Whether to remove the optix library after building chrono. This should be `'true'` if you plan to push the image to some public repository.
```

Comment:
why is this not default to true?

Reply:
Cause we can't push optix binaries/source, against the terms